services/user_service/database_connector.py

```diff
@@ -67,44 +65,49 @@ def save_user(self, user_info):
             print("###EROR received : " + str(e))
         except FirebaseError as e:
             print("###Connection lost. EROR received : " + str(e))
+        return None
 
-    
     def update_user(self, user_id, user_info):
         try:
             self.__users_ref.child(user_id).set(user_info)
+            return True
         except ValueError as e:
             print("###EROR received : " + str(e))
         except TypeError as e:
             print("###EROR received : " + str(e))
         except FirebaseError as e:
             print("###Connection lost. EROR received : " + str(e))
-    
+        return False
 
     def delete_user_by_id(self, user_id):
         try:
             self.__users_ref.child(user_id).delete()
+            return True
         except ValueError as e:
             print("###EROR received : " + str(e))
         except FirebaseError as e:
             print("###Connection lost. EROR received : " + str(e))
-
+        return False
 
     def delete_user_by_email(self, email):
         try:
             for i in self.get_user_ids():
                 if email == self.__users_ref.child(i).child("email").get():
                     self.delete_user_by_id(i)
+                    return True
         except ValueError as e:
             print("###EROR received : " + str(e))
         except FirebaseError as e:
             print("###Connection lost. EROR received : " + str(e))
-
+        return False
 
     def drop_users(self):
         try:
             for user_id in self.get_user_ids():
                 self.__users_ref.child(user_id).delete()
+            return True
         except ValueError as e:
             print("###EROR received : " + str(e))
```

Comment:
What is EROR?

Reply:
Except Rare Object Runtime? :)

just a typo

Reply:
You've repeated it so many times, that I thought there is sense in it